packages/theme/components/CartSidebar.vue

```diff
@@ -141,12 +161,20 @@ export default {
     const products = computed(() => cartGetters.getItems(cart.value));
     const totals = computed(() => cartGetters.getTotals(cart.value));
     const totalItems = computed(() => cartGetters.getTotalItems(cart.value));
-
+    const { wishlist, addItem: addItemToWishlist, isInWishlist } = useWishlist();
+    const isWishlistDisabled = computed(() => wishlistGetters.isWishlistDisabled(wishlist.value));
     const updateQuantity = debounce(async ({ product, quantity }) => {
       console.log('debug:updateQuantity', product, quantity);
       await updateItemQty({ product, quantity });
     }, 500);
 
+    const handleSaveForLaterClick = async(product) => {
+      if (!isInWishlist({product})) {
+        await addItemToWishlist({product});
+        await removeItem({product});
```

Comment:
Is there any reason those two Promises are awaited in such a manner (one after another)? If not, we could use `await Promise.all([addItemToWishlist({product}), removeItem({product})])`.